Mesh_3/include/CGAL/Mesh_3/Mesh_complex_3_in_triangulation_3_base.h

```diff
@@ -1049,14 +1049,15 @@ bbox(const double offset) const
     result = result + tr_.point(vit).bbox();
   }
 
-  const double offsetmin = 1. - offset;
-  const double offsetmax = 1. + offset;
-  return result = Bbox_3(offsetmin * result.xmin(),
-                         offsetmin * result.ymin(),
-                         offsetmin * result.zmin(),
-                         offsetmax * result.xmax(),
-                         offsetmax * result.ymax(),
-                         offsetmax * result.zmax());
+  const auto offset_x = offset * (result.xmax() - result.xmin()) / 2.;
+  const auto offset_y = offset * (result.ymax() - result.ymin()) / 2.;
+  const auto offset_z = offset * (result.zmax() - result.zmin()) / 2.;
```

Comment:
I am actually not sure about the `/ 2.`. In the current formulation, the xyz- lengths of the bbox are multiplied by `1 + offset`. Is that the effect you wanted? Or was it `1 + 2*offset`?